Backend/app/__main__.py

```diff
@@ -16,7 +16,16 @@
     allowed_origins,
     max_age,
 )
-from app.routers import artistas, canciones, generos, login, playlists, search, usuarios
+from app.routers import (
```

Comment:
Imports format shouldnt change. Ruff doesnt format the inline imports that appeared before.